Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.62 AS builder
+WORKDIR /tmp/
+COPY Cargo.toml Cargo.lock ./
+COPY src/lib.rs src/lib.rs
+
+# this build step will cache your dependencies
+RUN cargo build --release && rm -r src
+
+# copy your source tree
+COPY ./src ./src
+
+# build for release
+RUN cargo build --release
+
+FROM ubuntu:20.04
+RUN apt update && apt install -yy openssl ca-certificates
+COPY --from=builder /tmp/target/release/indexer-accounts .
```

Comment:
Looks like you copy pasted from the other project and forgot to change the names

Reply:
`indexer-accounts` is the name of the package? https://github.com/near/near-indexer-events/blob/main/Cargo.toml#L2

Reply:
`indexer-accounts` is the name of the package? https://github.com/near/near-indexer-events/blob/main/Cargo.toml#L2

Reply:
lol, thank you
That was my copy-paste error
I'll merge this, fix the names and send the PR with you as the reviewer